geom/src/polygon.rs

```diff
@@ -408,7 +407,7 @@ impl Polygon {
     /// Usually m^2, unless the polygon is in screen-space
     pub fn area(&self) -> f64 {
         // Polygon orientation messes this up sometimes
```

Comment:
Do you know if this comment is still applicable?

Reply:
876930d6e612b71bd1703d33554688bf54d72d3f